### PR TITLE
Brush up select Feat Effect descriptions and icons

### PR DESCRIPTION
**Pull request by ammalagonc**
Also, lint

### CODE REVIEW

File: packs/feat-effects/effect-fleet-tempo.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Fleet Tempo",
     "system": {
         "description": {
-            "value": "<p>You and all allies within 30 feet gain a +10-foot status bonus to all Speeds for 1 minute.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Fleet Tempo]</p>\n<p>You and all allies within 30 feet gain a +10-foot status bonus to all Speeds for 1 minute.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-fleet-tempo.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Fleet Tempo]</p>\n<p>You and all allies within 30 feet gain a +10-foot status bonus to all Speeds.</p>"
```

File: packs/feat-effects/effect-favorable-winds.json [hunk 2]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Favorable Winds",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Favorable Winds].</p>\n<hr />\n<p>You wrap a shell of favorable wind around a weapon you wield. On your next Strike with a ranged weapon before the start of your next turn, you don't take circumstance penalties on the attack roll due to wind. In addition, if that weapon has a range increment of 10 feet or less, increase its range increment by 10 feet for that Strike.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Favorable Winds].</p>\n<p>If your weapon has a range increment of 10 feet or less, increase its range increment by 10 feet for that Strike.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-favorable-winds.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Favorable Winds]</p>\n<p>If your weapon has a range increment of 10 feet or less, increase its range increment by 10 feet for that Strike.</p>"
```

File: packs/feat-effects/effect-devrins-cunning-stance.json [hunk 3]
```diff
@@ -4,10 +4,10 @@
     "name": "Effect: Devrin's Cunning Stance",
     "system": {
         "description": {
-            "value": "<p>You have a +1 status bonus to skill checks. When you Strike an @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] enemy, that enemy can't use reactions until the beginning of its next turn.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Devrin's Cunning Stance]</p>\n<p>You have a +1 status bonus to skill checks. When you Strike an @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] enemy, that enemy can't use reactions until the beginning of its next turn.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-devrins-cunning-stance.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Devrin's Cunning Stance]</p>\n<p>You have a +1 status bonus to skill checks. When you Strike an off-guard enemy, that enemy can't use reactions until the beginning of its next turn.</p>"
```

File: packs/feat-effects/effect-disruptive-stare.json [hunk 4]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Disruptive Stare",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Disruptive Stare]</p>\n<p>You take a –2 status penalty to its spell attack rolls and DCs.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-disruptive-stare.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Disruptive Stare]</p>\n<p>You take a –2 status penalty to your spell attack rolls and DCs.</p>"
```

File: packs/feat-effects/effect-distant-wandering.json [hunk 5]
```diff
@@ -4,10 +4,10 @@
     "name": "Effect: Distant Wandering",
     "system": {
         "description": {
-            "value": "<p>You are @UUID[Compendium.pf2e.conditionitems.Item.Invisible] and inaudible.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Distant Wandering]</p>\n<p>You are @UUID[Compendium.pf2e.conditionitems.Item.Invisible] and inaudible.</p>\n<p><strong>Awakening</strong> You gain a fly Speed and a +20-foot status bonus to your Speed</p>\n<p><strong>Awakening</strong> You are Quickened while in spirit form and can use the extra action only to Recall Knowledge or Seek.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-distant-wandering.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Distant Wandering]</p>\n<p>You are invisible and inaudible.</p>\n<p><strong>Awakening</strong> You gain a fly Speed and a +20-foot status bonus to your Speed</p>\n<p><strong>Awakening</strong> You are Quickened while in spirit form and can use the extra action only to Recall Knowledge or Seek.</p>"
```

File: packs/feat-effects/effect-duelists-challenge.json [hunk 6]
```diff
@@ -4,7 +4,7 @@
...
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Duelist's Challenge].</p>\n<p>Select one foe that you can see and proclaim a challenge. That foe is your dueling opponent until they are defeated, flee, or the encounter ends.</p>\n<p>Any time you hit that enemy using a single one-handed melee weapon while your other hand or hands are free, you gain a circumstance bonus to the Strike's damage equal to the number of damage dice your weapon deals.</p>\n<p>If you attack a creature other than your dueling opponent, you take a circumstance penalty to damage equal to the number of damage dice your weapon deals.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Duelist's Challenge].</p>\n<p>Any time you hit your dueling opponent using a single one-handed melee weapon while your other hand or hands are free, you gain a circumstance bonus to the Strike's damage equal to the number of damage dice your weapon deals.</p>\n<p>If you attack a creature other than your dueling opponent, you take a circumstance penalty to damage equal to the number of damage dice your weapon deals.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-duelists-challenge.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Duelist's Challenge]</p>\n<p>Any time you hit your dueling opponent using a single one-handed melee weapon while your other hand or hands are free, you gain a circumstance bonus to the Strike's damage equal to the number of damage dice your weapon deals.</p>\n<p>If you attack a creature other than your dueling opponent, you take a circumstance penalty to damage equal to the number of damage dice your weapon deals.</p>"
```

File: packs/feat-effects/effect-elemental-assault.json [hunk 7]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Elemental Assault",
     "system": {
         "description": {
-            "value": "<p>You shroud your arms and held weapons in elemental magic. Choose one element. Until the end of your next turn, your Strikes deal an additional 1d6 damage of the indicated type and have the trait corresponding to the element: electricity for air, bludgeoning for earth, fire for fire, or cold for water.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Elemental Assault]</p>\n<p>Your Strikes deal an additional 1d6 damage of the indicated type and have the trait corresponding to the element: electricity for air, bludgeoning for earth, fire for fire, or cold for water.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-elemental-assault.json:7]: Consider using `AdjustStrike` to add the trait to the strike.
See "Spell Effect: Inner Upheaval".

File: packs/feat-effects/effect-eye-of-the-arclords.json [hunk 8]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Eye of the Arclords",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Eye of the Arclords].</p>\n<p>While the eye is open, you gain the following benefits:</p>\n<ul>\n<li>you gain the effects of the detect magic arcane innate spell from your Arcane Sense at the start of each of your turns without needing to cast the spell;</li>\n<li>you gain darkvision;</li>\n<li>you gain a +2 status bonus to Perception checks to Seek undetected and hidden creatures and to your Perception DC against Hide and Sneak.</li>\n</ul>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Eye of the Arclords].</p>\n<p>You gain darkvision and a +2 status bonus to Perception checks to Seek undetected and hidden creatures and to your Perception DC against Hide and Sneak.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-eye-of-the-arclords.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Eye of the Arclords]</p>\n<p>You gain darkvision and a +2 status bonus to Perception checks to Seek undetected and hidden creatures and to your Perception DC against Hide and Sneak.</p>"
```

File: packs/feat-effects/effect-entitys-resurgence.json [hunk 9]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Entity's Resurgence",
     "system": {
         "description": {
-            "value": ""
+            "value": "<p>Granted by @UUID[Compendium.pf2e.actionspf2e.Item.Entity's Resurgence]</p>\n<p>You remain at 1 Hit Point and gain temporary Hit Points equal to your level + your key ability modifier. <span style=\"font-family:Roboto, sans-serif\">However, the entity is in control. </span>While the entity is in control, you gain a +1 status bonus to attack rolls and damage rolls</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-entitys-resurgence.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.actionspf2e.Item.Entity's Resurgence]</p>\n<p>You remain at 1 Hit Point and gain temporary Hit Points equal to your level + your key ability modifier. However, the entity is in control. While the entity is in control, you gain a +1 status bonus to attack rolls and damage rolls.</p>"
```

File: packs/feat-effects/effect-fleet-tempo.json [hunk 10]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Fleet Tempo",
     "system": {
         "description": {
-            "value": "<p>You and all allies within 30 feet gain a +10-foot status bonus to all Speeds for 1 minute.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Fleet Tempo]</p>\n<p>You and all allies within 30 feet gain a +10-foot status bonus to all Speeds.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-fleet-tempo.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Fleet Tempo]</p>\n<p>You gain a +10-foot status bonus to all Speeds.</p>"
```

File: packs/feat-effects/effect-energetic-meltdown.json [hunk 11]
```diff
@@ -9,11 +9,12 @@
...
+            "value": "<p>Granted by @UUID[Compendium.pf2e.classfeatures.Item.Energetic Meltdown]</p>\n<p><strong>Mild Backlash</strong> You take [[/r @actor.level]]{damage equal to your level} as energy scours your body.</p>\n<p><strong>Moderate Backlash</strong> After taking the damage, you gain weakness to the energy type of your backlash until your next daily preparations. The weakness is equal to one-third of your level. You also lose immunity to the energy type if you had it, until your next daily preparations.</p>\n<p><strong>Severe Backlash</strong> You deal [[/r (@actor.level)d6]]{1d6 damage per level} to all creatures in a @Template[emanation|distance:30] with a @Check[fortitude|against:class-spell|basic] save. You automatically take full damage without attempting a save.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-energetic-meltdown.json:17]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.classfeatures.Item.Energetic Meltdown]</p>\n<p><strong>Mild Backlash</strong> You take @Damage[(@actor.level)[untyped]]{damage equal to your level} as energy scours your body.</p>\n<p><strong>Moderate Backlash</strong> After taking the damage, you gain weakness to the energy type of your backlash until your next daily preparations. The weakness is equal to one-third of your level. You also lose immunity to the energy type if you had it, until your next daily preparations.</p>\n<p><strong>Severe Backlash</strong> You deal @Damage[(@actor.level)d6[untyped]]{1d6 damage per level} to all creatures in a @Template[emanation|distance:30] with a @Check[fortitude|against:class-spell|basic] save. You automatically take full damage without attempting a save.</p>"
```

File: packs/feat-effects/effect-devrins-cunning-stance.json [hunk 12]
```diff
@@ -4,10 +4,10 @@
     "name": "Effect: Devrin's Cunning Stance",
     "system": {
         "description": {
-            "value": "<p>You have a +1 status bonus to skill checks. When you Strike an @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] enemy, that enemy can't use reactions until the beginning of its next turn.</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Devrin's Cunning Stance]</p>\n<p>You have a +1 status bonus to skill checks. When you Strike an off-guard enemy, that enemy can't use reactions until the beginning of its next turn.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-devrins-cunning-stance.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Devrin's Cunning Stance]</p>\n<p>You gain a +1 status bonus to skill checks. When you Strike an off-guard enemy, that enemy can't use reactions until the beginning of its next turn.</p>"
```

File: packs/feat-effects/effect-crimson-shroud.json [hunk 13]
```diff
@@ -4,7 +4,7 @@
     "name": "Effect: Crimson Shroud",
     "system": {
         "description": {
-            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crimson Shroud].</p>"
+            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crimson Shroud].</p>\n<p>You gain fast healing equal to half your level. You gain the @UUID[Compendium.pf2e.actionspf2e.Item.Swirl Crimson Shroud] action.</p>"
```
Comment (simonward) [inline packs/feat-effects/effect-crimson-shroud.json:7]:
```suggestion
            "value": "<p>Granted by @UUID[Compendium.pf2e.feats-srd.Item.Crimson Shroud]</p>\n<p>You gain fast healing equal to half your level. You gain the @UUID[Compendium.pf2e.actionspf2e.Item.Swirl Crimson Shroud] action.</p>"
```

Comment (simonward) [reply packs/feat-effects/effect-crimson-shroud.json:7]: Consider removing empty `"details"`  from RE.

Comment (ammalagonc) [reply packs/feat-effects/effect-crimson-shroud.json]: ??
You mean empty space in the formula? that should be fixed now.

**Pull request merged**